fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/Plan.java

```diff
@@ -90,6 +91,16 @@ default boolean displayExtraPlanFirst() {
 
     List<Slot> getNonUserVisibleOutput();
 
+    default JSONObject toJson() {
+        JSONObject json = new JSONObject();
+        json.put("PlanType", getType().toString());
+        return json;
+    }
+
+    default Plan readFromJson(JSONObject jsonObject) {
+        return withLogicalProperties(null);
+    }
+
```

Comment:
It's redundant, we don't need add default, because we already have `AbstractPlan.java`, all plan already extends it.

Reply:
sure